### ISSUE TITLE
Cmap 'Blues' instead of 'Oranges'

### ISSUE THREAD

**Issue by pierrepo** [matplotlib/cheatsheets#42]
There might be a typo in the tips & tricks handout where a `Blues` color map is called although an `Oranges` one is displayed:

![image](https://user-images.githubusercontent.com/733861/91155892-b8b37000-e6c3-11ea-9737-d809f2616b73.png)

For consistency, maybe we could use
```
cmap = plt.get_cmap("Oranges")
```
instead of 
```
cmap = plt.get_cmap("Blues")
```

**Comment by rougier** [comment]
You're right, thanks. Can you make a PR (ese I can correct it directly)?

**Comment by jimustafa** [comment]
This appears to have been fixed in #24. The current (fixed) handout for tips & tricks is available at https://matplotlib.org/cheatsheets/handout-tips.pdf.
